### PR TITLE
Configure project to read environment variables

### PR DESCRIPTION
**Pull request by ghost**
## Summary

* Configure project to use env variables

## Screenshots

image

## Trello Card

[Trello Card](https://trello.com/c/RmS8To0N/71-n-94-api-ii-variables-de-ambiente)

### CODE REVIEW

File: wbooks/android/app/build.gradle [hunk 1]
```diff
@@ -217,3 +217,4 @@ task copyDownloadableDepsToLibs(type: Copy) {
 }
 
 apply from: file("../../node_modules/@react-native-community/cli-platform-android/native_modules.gradle"); applyNativeModulesAppBuildGradle(project)
+apply from: project(':react-native-config').projectDir.getPath() + "/dotenv.gradle"
```
Comment (sfernandez11) [inline wbooks/android/app/build.gradle]: Wrong line! Check docs again please

**Review by sfernandez11 — COMMENTED** [review]
Just one comment

File: wbooks/android/app/build.gradle (hunk 1 above)
Comment (ghost) [reply wbooks/android/app/build.gradle]: Done!

**Review by sfernandez11 — APPROVED** [review]
🥇

**Pull request merged**